devicetypes/smartthings/zigbee-accessory-dimmer.src/zigbee-accessory-dimmer.groovy

```diff
@@ -24,7 +24,7 @@ metadata {
 		capability "Health Check"
 
 		fingerprint profileId: "0104", inClusters: "0000,1000,0003", outClusters: "0003,0004,0005,0006,0008,1000,0019", manufacturer: "Aurora", model: "Remote50AU", deviceJoinName: "Aurora Wireless Wall Remote"
-		fingerprint manufacturer: "IKEA of Sweden", model: "TRADFRI wireless dimmer", deviceJoinName: "IKEA TRÅDFRI Wireless dimmer" // 01 [0104 or C05E] 0810 02 06 0000 0001 0003 0009 0B05 1000 06 0003 0004 0006 0008 0019 1000
+		fingerprint manufacturer: "IKEA of Sweden", model: "TRADFRI wireless dimmer", deviceJoinName: "IKEA TRÅDFRI Wireless dimmer", mnmn: "SmartThings", vid: "generic-dimmer" // 01 [0104 or C05E] 0810 02 06 0000 0001 0003 0009 0B05 1000 06 0003 0004 0006 0008 0019 1000
```

Comment:
If this device doesn't have a button it probably shouldn't be using this DTH with the `Button` capability

Reply:
@tpmanley On further review, I am going to revert the last PR and see if there is a better place for it. It might be better in the Zigbee battery accessory dimmer (though if I recall from the community, this device doesn't really report battery despite advertising it).